src/reference/changes.rst

```diff
@@ -107,6 +107,18 @@ makes Cylc 7 "reset to waiting" use cases much easier:
 For more information on Cylc IDs, run ``cylc help id``.
 
 
+Triggering tasks at initial and final cycle point
```

Comment:
This isn't just a trigger feature, suggest a more abstract title, maybe:

```suggestion
Selecting tasks at initial and final cycle point
```

?